osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -501,8 +503,12 @@ private void load(OsuColour colours, AudioManager audioManager, NotificationOver
                     notificationOverlay.Hide();
 
                     volumeOverlay.IsMuted.Value = false;
-                    audioManager.Volume.SetDefault();
-                    audioManager.VolumeTrack.SetDefault();
+
+                    // Check values before resetting, as the user may have only had mute enabled, in which case we might not need to adjust volumes.
+                    if (audioManager.Volume.Value < volume_requirement)
```

Comment:
This checks for `<` whereas the above code is checking for `<=`. I'd like them to match to prevent any FP weirdness.